packages/admin-ui/src/pages/wifi/components/WifiService.tsx

```diff
@@ -48,16 +50,18 @@ function WifiAlert({ wifiReport }: { wifiReport: WifiReport }) {
   return (
     <Alert
       dismissible
-      variant={wifiReport.containerState === "running" ? "success" : "warning"}
+      variant={
+        wifiReport.containerState === ("exited" || "dead") ? "warning" : "info"
+      }
     >
-      <Alert.Heading>Report:</Alert.Heading>
-      <p>
-        {wifiReport.info}.{" "}
-        <RenderMarkdown
-          source={`~~~js\n${wifiReport.report?.lastLog}\n~~~` || ""}
-        />
-      </p>
-      <hr />
+      <Alert.Heading>{wifiReport.info}</Alert.Heading>
+      {wifiReport.report ? (
+        <p>
+          <RenderMarkdown
```

Comment:
Why do we want to render markdown here?

Reply:
Last error log from the wifi container when `exited` or `dead`

I tried to be consistent since I cannot prevent every single log from wifi package. Also if hardcoding known exit logs it may be outdated in upcoming releases